RecoLocalCalo/HcalRecProducers/python/HBHEMethod2Parameters_cfi.py

```diff
@@ -11,7 +11,7 @@
     ts4Max                = cms.vdouble(100.,70000.), #fC # this is roughly 20 GeV
     pulseJitter           = cms.double(1.),           #GeV/bin 
     ###
-    meanTime              = cms.double(0.),   #ns 
+    meanTime              = cms.double(2.),   #ns 
```

Comment:
@slava77 
c847539 is a different initialization of the M2-fit meant to make the fit easier to converge and less CPU rather than introducing energy performance changes.  
This is what I learned from the plots I made on the single MC pion. 
This specific change was only discussed in the context of the phase1 thursday meetings (twice).
In the context of the 2016, at the DPG we discussed the MC timing and a current shift the digitization of the MC back to the center but the focus was the shift of the MC simulation (so the input of the M2) https://indico.cern.ch/event/561059/contributions/2269081/attachments/1320863/1980774/2016-08-05-Miao_HCAL_meeting_updata.pdf. 

In general we should bring the MC closer to data and not the other way around so best will to change the simulation rather than the reconstruction. 
Another option can be to have proper initial parameters for MC and data (i.e. meanTimeMC=2ns and dataMC=0ns) but this would also contrast with the current choise of using the pulse template derived from the data even for MC.

I will remove this modification (c847539) from this PR but point to the HCAL-DPG conveners.
Let me know if you think differently.